src/sst/core/configGraph.cc

```diff
@@ -30,6 +31,48 @@ namespace SST {
 
 // static bool zero_latency_warning = false;
 
+// Functions to check component and link names
+static int   bad_comp_name_count = 0;
+static int   bad_link_name_count = 0;
+static regex valid_name_regex =
+    regex("((([_][a-zA-Z0-9])|[a-zA-Z])[_a-zA-Z0-9]*)(.(([_][a-zA-Z0-9])|[a-zA-Z])[_a-zA-Z0-9]*)*");
+
+static const int max_invalid_name_prints = 10;
+
+bool
```

Comment:
If the goal is to eventually error in this function you could make it return `void` then when you are ready to error you can add the error code later.  

As written it looks like you want the caller of this function to read the `bool` and decide what to do about it, but that's not how the code that calls this is using it.